### ISSUE TITLE
Question GitLab CI pipelines dashboard

### ISSUE THREAD

**Issue by kaotu** [mvisonneau/gitlab-ci-pipelines-exporter#231]
Hi @mvisonneau,
This is lovely monitor for gitlab ci, but I have little bit problem about who commit code.
I want to display the user who commits code or trigger a pipeline. How I config `dashboard_pipelines.json` ?

**Comment by mvisonneau** [comment]
👋 hey @kaotu,

Unfortunately we do not currently export this information so you won't be able to get it in the dashboard yet.

**Comment by kaotu** [comment]
Hey @mvisonneau,
Thank for your answer. I have one question about `author_email` on  gitlab_ci_environment_information.
Why author_email is empty on prometheus metrics? 😀

```
# HELP gitlab_ci_environment_information Information about the environment
# TYPE gitlab_ci_environment_information gauge
gitlab_ci_environment_information{author_email="",available="true",current_commit_short_id="5f6581e2",environment="development",environment_id="26",external_url="",kind="branch",latest_commit_short_id="5f6581e2",project="gitlab101/golang",ref="develop"} 1
```

**Comment by mvisonneau** [comment]
Indeed @kaotu, thanks for highlighting this out! I opened a new issue regarding this specific problem. A fix is on its way!

**Comment by mvisonneau** [comment]
as the fix has now been released as part of [v0.4.8](https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/releases/tag/v0.4.8), I will close this issue!
